### ISSUE TITLE
[Feature Request]: Provide accessibleName prop to SideNavigationItem

### ISSUE THREAD

**Issue by volha-arkhipenka-sap** [SAP/ui5-webcomponents#11010]
### Feature Request Description

To be a11y compliant, every element should have a descriptive name, which is announced by a screenreader. However, the `text` of `SideNavigationItem` component could vary from what is expected to be announced by a screen reader and/or there could be a necessity to provide additional information.

### Proposed Solution

Add an `accessibleName` name prop to `SideNavigationItem` [component](https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/layouts-floorplans-sidenavigation--docs#sidenavigationitem) (e.g. similarly to [MenuItem](https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/modals-popovers-menu--docs#menuitem)), which will allow to programmatically specify the text, which is expected be read by a screen reader. 

### Additional Context

We've encountered a necessity to provide an amount counters for each `SideNavigationItem` (please, refer a screenshot). Despite the fact, that `SideNavigationItem` doesn't allow to provide anything except for pure string values to be displayed as an item's text, we've managed to add counters as another component, absolutely positioned over `SideNavigation`. 
However, there's still missing a11y support - as we couldn't provide custom `accessibleName` to `SideNavigationItem`, the counters are not pronounced by screen readers.

<img width="283" alt="Image" src="https://github.com/user-attachments/assets/d6d39994-7821-47e9-b6e9-53cc0dbd3c20" />

### Priority

Medium

### Privacy Policy

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Todor-ads** [comment]
Hello @SAP/ui5-webcomponents-topic-rd  ,
Could you take a look at this feature request?

Thanks,
Todor

**Comment by ArtemJJS** [comment]
Hello, are there any updates?

**Comment by AlekseiSch** [comment]
Hello, we're still waiting for this feature, do you have any news?

**Comment by LidiyaGeorgieva** [comment]
Hello, BLI for implementation was created under number 3639. We will keep you informed of progress.

**Comment by duygu-rmdn** [comment]
Hi @volha-arkhipenka-sap,

The issue has been fixed on the main branch in the following PR: https://github.com/UI5/webcomponents/pull/13402

Best, 
Duygu
